resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/config.scala

```diff
@@ -129,4 +129,22 @@ package object config {
         "when launching drivers. Default is to accept all offers with sufficient resources.")
       .stringConf
       .createWithDefault("")
+
+  private[spark] val MAX_DRIVERS =
+    ConfigBuilder("spark.mesos.maxDrivers").intConf.createWithDefault(200)
+
+  private[spark] val RETAINED_DRIVERS =
+    ConfigBuilder("spark.mesos.retainedDrivers").intConf.createWithDefault(200)
+
+  private[spark] val CLUSTER_RETRY_WAIT_MAX_SECONDS =
+    ConfigBuilder("spark.mesos.cluster.retry.wait.max")
+        .intConf
+        .createWithDefault(60) // 1 minute
+
+  private[spark] val ENABLE_FETCHER_CACHE =
+    ConfigBuilder("spark.mesos.fetcherCache.enable")
+        .doc("If set to true, all URIs (example: `spark.executor.uri`, `spark.mesos.uris`) will " +
+            "be cached by the Mesos Fetcher Cache.")
+        .booleanConf
+        .createWithDefault(false)
```

Comment:
Let's not add this. What we need is just the functional logic of SPARK-26192.

Reply:
So use magic strings instead of config objects?

Reply:
Of course, that's the backport.